jerry-core/ecma/builtin-objects/ecma-builtin-helpers.c

```diff
@@ -82,8 +82,8 @@ ecma_builtin_helper_object_to_string (const ecma_value_t this_arg) /**< this arg
 
   /* Building string "[object #type#]" where type is 'Undefined',
      'Null' or one of possible object's classes.
-     The string with null character is maximum 19 characters long. */
-  const lit_utf8_size_t buffer_size = 19;
+     The string with null character is maximum 27 characters long. */
+  const lit_utf8_size_t buffer_size = 27;
   JMEM_DEFINE_LOCAL_ARRAY (str_buffer, buffer_size, lit_utf8_byte_t);
```

Comment:
Perhaps we should change this to:

    lit_utf8_byte_t str_buffer[buffer_size];

I don't think malloc is needed here.